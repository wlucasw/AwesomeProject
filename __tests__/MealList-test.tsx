import 'react-native';
import React from 'react';
import renderer from 'react-test-renderer';
import MealList from '../mealList';

test('renders correctly', () => {
  const tree = renderer.create(<MealList />).toJSON();
  expect(tree).toMatchSnapshot();
});
