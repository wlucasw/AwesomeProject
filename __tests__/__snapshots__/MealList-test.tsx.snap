// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
    }
  }
>
  <Text>
    Apple Frangipan Tart
     : 
    British
     
    Dessert
  </Text>
  <Text>
    Apple & Blackberry Crumble
     : 
    British
     
    Dessert
  </Text>
  <Text>
    Apam balik
     : 
    Malaysian
     
    Dessert
  </Text>
  <Text>
    Ayam Percik
     : 
    Malaysian
     
    Chicken
  </Text>
</View>
`;
